src/components/experimental/Button/Button.tsx

```diff
@@ -87,15 +110,25 @@ const ButtonStyled = styled(BaseButton)<{ $emphasis: Emphasis }>`
         opacity: 0.24;
     }
 
+    &[data-pending] {
+        opacity: 0.38;
+    }
+
     ${textStyles.variants.label1}
 
     ${emphasisStyles};
 `;
 
-function Button({ children, emphasis = 'primary', ...restProps }: ButtonProps): ReactElement {
+const spinnerColor: Record<Emphasis, string> = {
+    primary: getSemanticValue('surface'),
```

Comment:
I think it should be the same as the text color on the button surface -- so 'on-accent' in this case

Reply:
Done! thanks for pointing that out